### PR TITLE
Recover "WPB-5204 Remove unused APNS_VOIP code (#3695)"

### PR DESCRIPTION
**Pull request by fisx**
https://wearezeta.atlassian.net/browse/WPB-6583

This reverts commit 133a7409f152a60611eec9d97db11feb6551aa3a and fixes 66b8c98a83d5a48e75accb3c65a2ff36e28eb5be.

## Checklist

 - [ ] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [ ] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

File: libs/wire-api/test/unit/Test/Wire/API/MLS.hs [hunk 1]
```diff
@@ -308,7 +308,7 @@ spawn cp minput = do
        in snd <$> concurrently writeInput readOutput
   case (mout, ex) of
     (Just out, ExitSuccess) -> pure out
-    _ -> assertFailure "Failed spawning process"
+    _ -> assertFailure $ "Failed spawning process\n" <> show mout <> "\n" <> show ex
```
Comment (stefanwire) [inline libs/wire-api/test/unit/Test/Wire/API/MLS.hs:311]: Is this related to the current PR or a drive-by debug improvement?

**Review by stefanwire — APPROVED** [review]

File: libs/wire-api/test/unit/Test/Wire/API/MLS.hs (hunk 1 above)
Comment (fisx) [reply libs/wire-api/test/unit/Test/Wire/API/MLS.hs:311]: the latter (but it's small enough that it didn't bother me too much).
